### DISCUSSION TITLE
UserData from Official WSA

### DISCUSSION THREAD

**Discussion by IaemTary** [LSPosed/MagiskOnWSALocal#678] [Q&A]
I recently followed the instruction on creating a backup of userdata.vhdx file, I did by just copying it to another folder, then after shifting to this version of **WSA**, I placed the file back to same location, but doing so it corrupts the current apps to not open and makes **WSA** to behave weirdly, so how can I restore that userdata without causing issues, there were already many apps in the previous official version with over 13GB+ of data.

**Comment by Almighty188** [comment]
What version was the WSA that worked originally?  

This Windows app called WSATools allows backing up the WSA apps and data which was what I used as I had the same problem:
https://apps.microsoft.com/store/detail/wsatools-apk-installer-and-more/9N4P75DXL6FG?hl=en-us&gl=us&rtc=1

**Reply by IaemTary** [reply]
It was the official version.

**Reply by Almighty188** [reply]
I mean the actual version # as the userdata in 2306 changed to a different format so if the previous userdata was from a version older than 2306, it will not work for that reason.
